Gemfile.lock

```diff
@@ -1,22 +1,24 @@
 PATH
   remote: .
   specs:
-    sablon (0.0.21)
```

Comment:
Hadn't been bundled last time it was updated; `lib/sablon/version.rb` was `0.0.22` but this was `0.0.21`.